scripts/run.sh

```diff
@@ -73,19 +73,19 @@ RUN_TRANSIT=true
 # parse args
 while [ "$1" != "" ]; do
     case $1 in
-        -h | --help )           usage
-                                exit
-                                ;;
-        -p | --prod )           PROD=true
-                                ;;
-        -d | --dev )            DEV=true
-                                ;;
-        -t | --test )           TEST=true
-                                ;;
-        -s | --setup )          RUN_TRANSIT=false
-                                ;;
-        * )                     usage
-                                exit 1
+        -h | --help )                   usage
+                                        exit
+                                        ;;
+        -p | --prod | --production )    PROD=true
+                                        ;;
+        -d | --dev | --development )    DEV=true
```

Comment:
not the biggest fan of having two flags for `--dev` and `--development`, what's the reasoning behind not having just one?